pkg/chunk/cache/background.go

```diff
@@ -77,8 +77,8 @@ func (c *backgroundCache) Stop() error {
 	return c.Cache.Stop()
 }
 
-// StoreChunk writes chunks for the cache in the background.
-func (c *backgroundCache) StoreChunk(ctx context.Context, key string, buf []byte) error {
+// Store writes chunks for the cache in the background.
```

Comment:
Document nit: Since the goal with this change is to include index caching through the same methods, can you please update the function header comments to reflect this?